src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public String getRandomBall() {
+        int ballNumber = new Random().nextInt(100);
```

Comment:
lets make Random and ColorSupplier class level variables to avoid creating new instances each time this method is invoked

lets add a constant for 100 - avoid magic numbers